src/utils/stringUtil.ts

```diff
@@ -5,3 +5,15 @@ export const uppercaseFirstLetter = (word: string): string => {
 export const capitalizeHyphenatedWords = (word: string): string => {
   return word.split('-').map(uppercaseFirstLetter).join('-');
 };
+
+export const formatPersonname = (navn: string): string => {
```

Comment:
Kunne funksjonen fortalt oss noe om hva den formatter? 😊 `toFullName(…)` for eksempel 🤔
```suggestion
export const formatPersonname = (navn: string): string => {
```

Reply:
`toLastnameFirstname` kanskje 😬 Siden den flipper om navn og legger komma mellom.